x-pack/plugins/apm/public/components/routing/templates/apm_service_template/index.tsx

```diff
@@ -276,7 +276,22 @@ function useTabs({ selectedTab }: { selectedTab: Tab['key'] }) {
         path: { serviceName },
         query,
       }),
-      append: <BetaBadge icon="beaker" />,
+      append: (
+        <EuiBetaBadge
```

Comment:
Why don't use the `BetaBadge` reusable component we created?https://github.com/elastic/kibana/blob/af3a3cba9a7fa3b3e0d2e4766a1db11a5ff4f192/x-pack/plugins/apm/public/components/shared/beta_badge.tsx
This way we just need to pass the icon as a prop and reuse the copy that should be the same every time we need a beta badge

Reply:
What is the difference between the `BetaBadge` and the `EuiBetaBadge`?

Reply:
The BetaBadge is a reusable component that renders the EuiBetaBadge, using this reusable component helps so, any changes can be done in just one place, ie. we change the Beta copy in the tooltip, instead of needed to change in all the places we use the EuiBetaBadge

Reply:
Thank you I understand, and that makes me wonder if other teams in Kibana would need a different thing, and then the need of that component on top of component usage. 

BTW I've updated it to use the `BetaBadge` and added a title prop to the component too.  Thank you Miriam 🎉